test/behaviors/conference.behavior.js

```diff
@@ -533,6 +532,121 @@ function shouldBehaveLikeConference () {
       assertBalanceWithDeposit((await getBalance(conference.address)), mulBN(deposit, 0))
     })
   })
+
+  describe('on clear and send', function(){
+    let conference, deposit, attended, payoutAmount, clearFee, fees;
+
+    beforeEach(async function(){
+      conference = await createConference({coolingPeriod:0})
+      deposit = await conference.deposit()
+      
+      let numRegistered = 4;
+      for (let i = 0; i < numRegistered; i++) {
+          await register({conference, deposit, user:accounts[10 + i], owner})
+      } 
+
+      // [ 110 ], accounts[11], accounts[12]
+      const maps = [ toBN(0).bincn(1).bincn(2) ];
+
+      await conference.finalize(maps, {from:owner});
+      await conference.ended().should.eventually.eq(true)
+      
+      payoutAmount = new EthVal(await conference.payoutAmount());
+      clearFee = await conference.clearFee();
+      fees = payoutAmount.mul(clearFee).div(1000).toString(10);
+
+      assertBalanceWithDeposit((await getBalance(conference.address)), mulBN(deposit, 4))
+    })
+
+    it('fees calculation works correctly', async function(){
+      let previousBalanceOne = await getBalance(accounts[11]);
+      let previousBalanceTwo = await getBalance(accounts[12]);
+      let previousBalanceOwner = await getBalance(owner);
+
+      await wait(20, 1);
+      await conference.clearAndSend({from:owner});
+
+      let diffOne = new EthVal(await getBalance(accounts[11])).sub(previousBalanceOne)
+      let diffTwo = new EthVal(await getBalance(accounts[12])).sub(previousBalanceTwo)
+      let diffOwner = new EthVal(await getBalance(owner)).sub(previousBalanceOwner)
+
+      assert.isOk(diffOne.eq(payoutAmount.sub(fees)))
+      assert.isOk(diffTwo.eq(payoutAmount.sub(fees)))
+      assert.isOk(diffOwner.gt(new EthVal(fees).mul(1.9)))
```

Comment:
Does `1.9` means `(fees * 2 ) - gas`? can you either put comment or put that figure into a meaningful variable?

Reply:
Yes! Replaced that with `diffOwner <= (fees * 2)`